cli/context.go

```diff
@@ -40,6 +41,14 @@ func (ctx *Context) Slice(name string) []string {
 	return ctx.get(name).([]string)
 }
 
+func (ctx *Context) Int(name string) int {
+	value, err := strconv.Atoi(ctx.get(name).(string))
+	if err != nil {
+		panic(fmt.Errorf("interface conversion: interface is string, not int"))
```

Comment:
I don't really like panicking on user input, would it be too weird to return an error for Int if the others don't?